great_expectations/data_context/util.py

```diff
@@ -163,7 +163,7 @@ class PasswordMasker:
     MASKED_PASSWORD_STRING = "***"
 
     # values with the following keys will be processed with cls.mask_db_url:
-    URL_KEYS = {"connection_string", "url"}
+    URL_KEYS = {"conn_str", "connection_string", "url"}
```

Comment:
I wonder why we have `conn_str` and `connection_string`? Are they used in different data sources?